docker-compose.yml

```diff
@@ -28,9 +28,15 @@ services:
       - DATABASE_URL=mysql2://foreman:foreman@db:3306/foreman
       - RAILS_MAX_THREADS=5
       - RAILS_ENV=production
+      - FOREMAN_FQDN=foreman.example.com
```

Comment:
For another PR: Since you're setting FQDN and DOMAIN, you don't need facter to be installed. Optional optimization to not install those in the docker image and always require the user to set them. Probably nice to provide a hard error `facter is unavailable, please configure FQDN and domain` (or similar).

Reply:
@ekohl https://github.com/theforeman/foreman/pull/6838